<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="mapGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>601</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {

border: 3px solid blue;
border-radius: 6px;
padding: 0 8px;
background : lightyellow;

}</string>
   </property>
   <property name="title">
    <string>Rover's map</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>561</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mapGridLayout"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="inputGroupBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>131</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {

border: 1px solid blue;
border-radius: 6px;
padding: 0 8px;
background : lightgreen;

}</string>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QCheckBox" name="checkBox_IsRandomFill">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Fill random</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GenMap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

border: 2px solid gray;
border-radius: 6px;
padding: 0 8px;
background : blue;
color: white;

}</string>
    </property>
    <property name="text">
     <string>Generate map</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_N">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>28</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

color : blue;

}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_M">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>28</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

color : blue;

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelM">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>M:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>31</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>N:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Min">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

color : blue;

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Max">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

color : blue;

}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_XFreq">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

color : blue;

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelXFreq">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>X frequency 1 to :</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="outputGroupBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>131</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {

border: 1px solid red;
border-radius: 6px;
padding: 0 8px;
background : yellow;

}</string>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLabel" name="label_Fuel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string> Fuel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Steps">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Steps:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Steps">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

color : red;

}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Fuel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

color : red;

}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Solve">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

border: 2px solid gray;
border-radius: 6px;
padding: 0 8px;
background : red;
color: white;

}</string>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
